pidng/core.py

```diff
@@ -131,6 +122,96 @@ def parseMaker(s):
     return d
 
 
+PROFILE_EMBED = 3
+
+@dataclass
+class Profile:
+    name : None
+    ccm1 : None
+    ccm2 : None
+    illu1 : None
+    illu2 : None
+    as_shot_neutral : list
+    profile_name : None
+
+    def write(self, main_ifd, maker_note):
+        baseline_exp = 1
+
+        camera_calibration = [[1, 1], [0, 1], [0, 1],
```

Comment:
Is there any time where we might want to pass this information in when creating a new profile? We can always subclass the `write` function, but for any data like this that could be feasibly changed, it would make more sense to take it as an optional parameter.